app.js

```diff
@@ -1,30 +1,133 @@
 import express from "express";
 import nunjucks from "nunjucks";
-import axios from "axios";
+import mongoose from "mongoose";
+import * as uuid from "uuid";
+import dotenv from "dotenv";
+import fs from "fs";
+import filterGameOptions from "./server/js/filterGameOptions.js";
+import wordCheck from "./server/js/wordCheck.js";
+import Game from "./server/models/Game.js";
+import HighScore from "./server/models/highScore.js";
+import saveInDb from "./server/js/saveInDB.js";
+import getOneFromDB from "./server/js/getOneFromDB.js";
+import updateOneFromDB from "./server/js/updateOneFromDB.js"
+import highScoreRouter from "./server/js/highScoreRouter.js";
+
+dotenv.config();
 
 const app = express();
 
-const env = nunjucks.configure('views', {
-    autoescape: true,
-    express: app
+const DBUrl = process.env.DB_URL;
+
+const env = nunjucks.configure("views", {
+  autoescape: true,
+  express: app,
 });
 
-app.set('view engine', 'html');
+app.set("view engine", "html");
 
 app.use(express.static("./public"));
 
-app.get("/", (req,res)=> {
-    res.render("index.html");
+app.use(express.json());
+
+app.use("/high-scores", highScoreRouter)
```

Comment:
How do you decide whether a route gets it's own router, or is defined inline in this module (app.js)? To me it seems arbitrary to have this be defined in another file, while several other long routes are defined here in this file.

I think it's a good idea to split things up, but I think it should be consistent or at least predictable.